src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+/*eslint-disable */
+class Calculator extends Component {
+  render() {
+    return (
+      <div className="container">
+        <div className="wrapper">
```

Comment:
- [ ] Kindly match the given design and place the `%` and `+/-`  buttons accordingly as seen from the design.

Your Implementation | Design guideline
:---:|:---:
![](https://i.imgur.com/OlE2vzA.png) | ![](https://i.imgur.com/ovF5bmv.png)